mobile/lib/modules/sharing/providers/suggested_shared_users.provider.dart

```diff
@@ -1,10 +1,16 @@
 import 'package:hooks_riverpod/hooks_riverpod.dart';
-import 'package:immich_mobile/shared/models/user.model.dart';
 import 'package:immich_mobile/shared/services/user.service.dart';
+import 'package:openapi/api.dart';
 
 final suggestedSharedUsersProvider =
-    FutureProvider.autoDispose<List<User>>((ref) async {
+    FutureProvider.autoDispose<List<UserResponseDto>>((ref) async {
   UserService userService = ref.watch(userServiceProvider);
 
-  return await userService.getAllUsersInfo();
+  var users = await userService.getAllUsersInfo(isAll: false);
+
+  if (users != null) {
```

Comment:
`return await userService.getAllUsersInfo(isAll: false) ?? [];` would be clearer IMO